documentation/guide-kafka.asciidoc

```diff
@@ -6,393 +6,7 @@ toc::[]
-----
-<dependency>
-  <groupId>com.devonfw.java.starters</groupId>
-  <artifactId>devon4j-starter-kafka-receiver</artifactId>
-</dependency>
-----
-
-==  Property Parameters
-
-As written before kafka-producer and listener-specific configuration is done via properties classes. These classes provide useful defaults, at a minimum the following parameters have to be configured:
-
-[source,properties]
-----
-messaging.kafka.common.bootstrap-servers=kafka-broker:9092
-messaging.kafka.consumer.group-id=<e.g. application name>
-messaging.kafka.listener.container.concurrency=<Number of listener threads for each listener container>
-----
-
-All the configuration beans for devon4j-kafka are annotated with `@ConfigurationProperties` and use common prefixes to read the property values from `application.properties` or `application.yml`.
-
-Example:
-
-[source,java]
-----
- @Bean
-  @ConfigurationProperties(prefix = "messaging.kafka.producer")
-  public KafkaProducerProperties messageKafkaProducerProperties() {
-    return new KafkaProducerProperties();
-  }
-----
-
-For producer and consumer the prefixes are `messaging.kafka.producer...` and `message.kafka.consumer...`` and for retry the prefix is `messaging.retry...``
-
-See https://github.com/devonfw-sample/devon4j-kafka-employeeapp/blob/master/core/src/main/resources/application.properties[devon4j-kafka-employeeapp's application.properties] file for an example.
-
-We use the same properties defined by Apache Kafka or Spring Kafka. They are simply "mapped" to the above prefixes to allow easy access from your application properties. The java docs provided in each of the devon4j-kafka property classes which explains their use and what value has to be passed.
-
-* The `https://github.com/devonfw/devon4j/blob/develop/modules/kafka/src/main/java/com/devonfw/module/kafka/common/messaging/api/config/KafkaProducerProperties.java/[Devon4j-Kafka-Producer properties]` are defined from `https://kafka.apache.org/documentation/#producerconfigs/[kafka prodcuer config]`.
-* The `https://github.com/devonfw/devon4j/blob/develop/modules/kafka/src/main/java/com/devonfw/module/kafka/common/messaging/api/config/KafkaConsumerProperties.java/[Devon4j-Kafka-Consumer properties]` are defined from `https://kafka.apache.org/documentation/#consumerconfigs/[kafka consumer config]`.
-* The `https://github.com/devonfw/devon4j/blob/develop/modules/kafka/src/main/java/com/devonfw/module/kafka/common/messaging/api/config/KafkaListenerContainerProperties.java/[Devon4j-Kafka-Listener container properties]` are defined from `https://docs.spring.io/spring-kafka/api/org/springframework/kafka/listener/ContainerProperties.html/[Spring kafka listener properties]`.
-
-
-== Naming convention for topics
-
-For better managing of several Kafka topics in your application portfolio we devon strongly advice to introduce a naming scheme for your topics. The schema may depend on the actual usage pattern of kafka. For context where kafka is used
```

Comment:
Remove the "devon" in "... we devon strongly advice ..."

Reply:
Kafka